pkg/agent/cniserver/ipam/annotations.go

```diff
@@ -15,6 +15,9 @@
 package ipam
 
 const (
-	AntreaIPAMAnnotationKey       = "ipam.antrea.io/ippools"
+	// AntreaIPAMAnnotationKey annotation can be added to Namespace and Template of StatefulSet/Deployment
```

Comment:
**PodTemplate** of StatefulSet/Deployment to be clear?

Reply:
Sure. Fixed.